transform/instance_test.go

```diff
@@ -226,6 +226,7 @@ func TestArrayTransform(t *testing.T) {
 				format:   jsonInput,
 				childTransformer: &objectTransformer{
 					jsonPath: "$.otherCrops[*][*]",
+					format:   jsonInput,
```

Comment:
Is there a similar test for XML?

Reply:
off handedly its covered in the transformer_test.go , but maybe it would be worth building some explicitly in here as well